modules/tournament/src/main/TournamentStandingApi.scala

```diff
@@ -56,8 +56,8 @@ final class TournamentStandingApi(
       compute(tourId, page, withScores = true)
 
   def clearCache(tour: Tournament): Unit =
-    first.invalidate(tour.id)
-    // no need to invalidate createdCache, these are only cached when tour.isCreated
+  first.invalidate(tour.id)
+  // no need to invalidate createdCache, these are only cached when tour.isCreated
```

Comment:
this looks strange, I would expect the implementation indentation to be unchanged here?